css/main.css

```diff
@@ -0,0 +1,1108 @@
+}
+
+.my__skills__top {
+  align-items: center;
+  justify-content: space-between;
+  padding: 23px 40px 23px 0;
+  cursor: pointer;
+  display: flex;
+}
+
+.my__skills__top__title {
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.clr-n-900 {
+  color: #000;
+}
+
+.clr-n-800 {
+  color: #172b4d;
+}
+
+.clr-n-700 {
+  color: #253858;
+}
+
+.clr-n-600 {
+  color: #344563;
+}
+
+.clr-n-500 {
+  color: #7a869a;
+}
+
+.clr-p-500 {
+  color: #396df2;
+}
+
+.clr-p-400 {
+  color: #6070ff;
+}
+
+.clr-p-300 {
+  color: #7f8cff;
+}
+
+input::placeholder {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+}
+
+hr {
+  position: absolute;
+  width: 134px;
+  height: 5px;
+  left: calc(50% - 134px / 2 - 0.5px);
+  bottom: 8px;
+  background: #0d151e;
+  border-radius: 10px;
+  visibility: hidden;
+}
+
+.form-section {
+  background-color: #fff;
+  background-image:
+    url("../statics/images/ep1.svg"),
+    url("../statics/images/rp1.svg"),
+    url("../statics/images/ep2.svg"),
+    url("../statics/images/rp2.svg"),
+    url("../statics/images/ep3.svg"),
+    url("../statics/images/contact-form-\ background.svg");
+  background-repeat: no-repeat;
+  background-position:
+    78% 91%,
+    86% 90%,
+    99.7% 9.2%,
+    99.5% 6.2%,
+    41% 7.06%,
+    0;
+  background-size: auto, auto, auto, auto, auto, cover;
+  display: flex;
+  justify-content: center;
+  margin-top: 100px;
+}
+
+.div-wrap {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 100px 24px 0 24px;
+  text-align: center;
+}
+
+.form_t {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+  margin-bottom: 20px;
+}
+
+.form-paragraph {
+  font-family: Poppins, sans-serif;
+  font-size: 20px;
+  line-height: 28px;
+  font-weight: 400;
+  text-align: center;
+  color: #ebebff;
+  margin: auto 15.5%;
+  padding: 0 14.5%;
+  font-style: normal;
+  margin-bottom: 50px;
+}
+
+.div-wrap .form-paragraph { margin-bottom: 24px; }
+
+.reach_me input {
+  font-family: Poppins, sans-serif;
+  font-size: 17px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  padding: 15px 10px;
+  border-radius: 12px;
+  border: none;
+}
+
+.reach_me textarea {
+  font-family: Poppins, sans-serif;
+  font-size: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+  padding: 15px;
+  border-radius: 12px;
+  border: none;
+}
+
+.reach_me .button {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  border: none;
+  width: 142px;
+  height: 48px;
+  padding: 12px 16px;
+  border-radius: 12px;
+  flex-direction: row;
+  justify-content: center;
+  margin: 24px auto;
+}
+
+@media (min-width: 721px) {
```

Comment:
Nice work styling our project😉😉 but let's remember that we are [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md)  to stick to `min-width: 768px` in this [milestone](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md)😉 

<img width="742" alt="minwidth media query" src="https://user-images.githubusercontent.com/64643995/221371853-277de248-4b0a-4c63-bed0-94f54a5b6b97.png">

Reply:
ok, I have fixed it!